### DISCUSSION TITLE
bind_tweets error : Error: Can't rename columns that don't exist. x Column `id` doesn't exist.

### DISCUSSION THREAD

**Discussion by shmuhammadd** [cjbarrie/academictwitteR#237] [Q&A]
Hi, 

I am not sure exactly why this. But, I collected tweets and one collection works well and the other collection reports an error when I try using `bind_tweets()` with `output_format = "tidy"`.

However, it works perfect without` output_format = "tidy"`, that is using `bind_tweets(data_path = "data/")`

The report is : 

```
Error: Can't rename columns that don't exist.
x Column `id` doesn't exist.
Run `rlang::last_error()` to see where the error occurred.
```

This works :` bind_tweets(data_path = "data/")`, and this reports an error : `bind_tweets(data_path = "data/", output_format = "tidy")`
 

Thanks.

**Comment by chainsawriot** [comment]
@shmuhammad2004 Once again, we need your data (at least the part that can reproduce this error). Otherwise, it's impossible to investigate.
